src/eventSchemas/templates/schemasAppTemplateUtils.ts

```diff
@@ -28,31 +28,31 @@ export interface SchemaTemplateParameters {
 
 // This matches the extra_content parameters to Schema-based templates in both key and value names in cookiecutter.json in the templates used by SamEventBridgeSchemaAppPython
 export interface SchemaTemplateExtraContext {
-    AWS_Schema_registry: string
-    AWS_Schema_name: string
-    AWS_Schema_root: string
-    AWS_Schema_source: string
-    AWS_Schema_detail_type: string
-    user_agent: string
+    AWS_Schema_registry: string // eslint-disable-line @typescript-eslint/naming-convention
```

Comment:
Assuming for now that there is a technical need for how these members are spelled.